common/browser.go

```diff
@@ -62,6 +62,9 @@ type Browser struct {
 	sessionIDtoTargetIDMu sync.RWMutex
 	sessionIDtoTargetID   map[target.SessionID]target.ID
 
+	// Used to display a warning when the browser is reclosed.
+	closed bool
```

Comment:
I would use `atomic.Bool` and then I changed my mind as a single VU enters here.